Userland/Libraries/LibWeb/WebDriver/TimeoutsConfiguration.cpp

```diff
@@ -48,42 +48,42 @@ ErrorOr<TimeoutsConfiguration, Error> json_deserialize_as_a_timeouts_configurati
     // 3. If value has a property with the key "script":
     if (value.as_object().has("script"sv)) {
         // 1. Let script duration be the value of property "script".
-        auto const& script_duration = value.as_object().get_deprecated("script"sv);
+        auto const& script_duration = value.as_object().get("script"sv);
```

Comment:
This shouldn't have a `const&`.

Kind of unfortunate the spec allows for a number or null :unamused: This could be so much simpler if it only allowed numbers